.github/workflows/label.yaml

```diff
@@ -0,0 +1,16 @@
+name: 🏷️ Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 1
+          labels: "kind/feature, kind/bug, kind/documentation, kind/test, kind/cleanup, dependabot"
```

Comment:
Should the last one be dependencies instead? That is the one dependabot automatically adds